jpa-crud/jpa-crud-java-sbt/jpa-crud-java-sbt-impl/src/main/resources/application.conf

```diff
@@ -0,0 +1,25 @@
+play.modules.enabled += com.example.jpacrud.EntityModule
+
+play.evolutions.db.default.autoApply = true
+
+jpa.default = defaultPersistenceUnit
+
+db.default {
+  driver = org.h2.Driver
+  url = "jdbc:h2:mem:test;MODE=Oracle;INIT=CREATE SCHEMA IF NOT EXISTS ENTITY"
+  jndiName = DefaultDS
+
+  hikaricp {
+    poolName: "entity.database"
+    connectionTimeout = 1s
+    connectionTestQuery = "SELECT 1 FROM DUAL"
+  }
+}
+
+database.dispatcher {
+  executor = "thread-pool-executor"
+  throughput = 1
+  thread-pool-executor {
+    fixed-pool-size = 10
+  }
+}
```

Comment:
This is the most important aspect of the recipe. All blocking code must run on a separate dispatcher which must be a thread pool (not a fork join!) as you demo here. It'd be great if that important details was documented here or in the README.md.

Reply:
Added info about separate dispatcher (here and in the README). 
If this is not enough, let me know 😉